packages/@sfx/sayt-driver-plugin/src/sayt-driver-plugin.ts

```diff
@@ -63,11 +65,29 @@ export default class SaytDriverPlugin implements Plugin {
     fields: ['*'],
   }
 
-  constructor() {
+  /**
+   * Default product transformer identity function.
+   * Intended to be overwritten by passing a custom product transformer.
+   *
+   * @param product The product to be returned as-is.
+   * @returns The received product object.
+   */
+  transformProduct: ProductTransformer<P> = ((product: Record): Record => product) as any;
```

Comment:
Yes, and since this is a property, we can change the documentation to be more generic:

```js
  /**
   * The product transformer to use that will transform a [[Record]] to
   * the desired form.
   */
```